projects/tags-input-demo/src/app/users-list-item/users-list-item.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="item" >
```

Comment:
The indentation in this file seems very off.
It seems like 4 spaces. Please make sure you have the `.editorconfig` extension installed for the VSCode and then format it using Ctrl K + Ctrl F (Cmd K + Cmd F for mac)

Reply:
fixed